src/pages/search/search.styles.js

```diff
@@ -0,0 +1,44 @@
+import styled from 'styled-components';
+
+export const Container = styled.div`
+  align-items: center;
+  display: flex;
+  flex-direction: column;
+  height: calc(100vh - 200px);
+`;
+
+export const Form = styled.form`
+  position: relative;
+  top: -40px;
```

Comment:
Maybe setting another bottom margin would be better than using a negative top value!?